src/views/Display/index.jsx

```diff
@@ -10,7 +10,9 @@ import Dashboard from '../../components/Dashboard';
 import DataTable from '../../components/DataTable';
 import { VNC_DISPLAYS_POLLING_INTERVAL } from '../../utils/constants';
 import ErrorPanel from '../../components/ErrorPanel';
+import withAlertOnClose from '../../utils/withAlertOnClose';
 
+@withAlertOnClose
```

Comment:
Remove the decorator from this view and insert it in the `VncViewer` component instead.